projects/animesite/index.html

```diff
@@ -0,0 +1,392 @@
+            <h4>Jujutsu Kaisen T-shirt </h4>
+            <div class="rating">
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star-o"></i>
+            </div>
+            <p>$25.90</p>
+        </div>
+        <div class="col4">
+            <div class="demi">
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT0OPdW7w_bsvH-kQSO5sqZPPU1LHGfY6l2Y6qIrtN7iek6Tsj30NyWmLQgdNr8InI-3y4&usqp=CAU"
+                    height="252px">
+            </div>
+            <h4>Wrist band from DS</h4>
+            <div class="rating">
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star"></i>
+                <i class="fa fa-star-o"></i>
+            </div>
+            <p>$40.10</p>
+        </div>
+    </div>
+    </div>
+
+    <div class="offer">
+        <div class="small-container"></div>
+        <div class="rows">
+            <div class="col-2">
+                <img class="offer-img"
+                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSw_cnBJrtiBAn8Zs_kIzOpZQzUgrfBUYmJjRzlAmlkTvkTJlIAQ-OKEnl_ZBKSuzkxi4g&usqp=CAU">
+
+            </div>
+
+            <div class="col-2">
+                <p>Exclusively Available On <strong><u>Addicted to Anime Website</u></strong></p>
+                <h2>Death Note Character "L" </h2>
+                <small>He is an enigmatic, mysterious, and highly-esteemed
+                </small><br>
+                <a href=""class="bt">Buy Now &#8594;</a>
+            </div>
+
+        </div>
+    </div>
+    <!-- ------testimonial------ -->
+    <div class="testimonial">
+        <div class="small-container">
+            <div class="rows">
+                <div class="col-3">
+                    <i class="fa fa-quote-left"></i>
+                    <p>Eiichiro Oda is a Japanese manga artist and the creator of the series One Piece. With more than
+                        500 million tankōbon copies in circulation worldwide, One Piece is both the best-selling manga
+                        in history and the best-selling comic series printed in volume, in turn making Oda one of the
+                        best-selling fiction authors..</p>
+                    <div class="rating">
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                    </div>
+                    <img
+                        src="http://t1.gstatic.com/licensed-image?q=tbn:ANd9GcQ-jN__eNpfqVbU1k-kifLJ45o1ht9mSneMDBmtX9G7ynCfb4uV-Cs6R4Upnnd4i3Em">
```

Comment:
This link is causing mixed content conflict (https/http) please change it with some other source

Reply:
@omk-coder